include/libcyphal/transport/udp/udp_transport_impl.hpp

```diff
@@ -388,6 +375,16 @@ class TransportImpl final : private TransportDelegate, public IUdpTransport  //
         return opt_any_error;
     }
 
+    void onSessionEvent(const SessionEvent::Variant& event_var) override
+    {
+        cetl::visit(
+            [this](const auto& event) {
+                SessionEventHandler handler_with{*this};
+                cetl::visit(handler_with, event);
```

Comment:
```suggestion
                cetl::visit(cetl::pf17::make_overloaded(
                                [this](const SessionEvent::Message::Destroyed& event) {
                                    msg_rx_session_nodes_.removeNodeFor(event.subject_id);  //
                                },
                                [this](const SessionEvent::Request::Destroyed& event) {
                                    svc_request_rx_session_nodes_.removeNodeFor(event.service_id);
                                },
                                [this](const SessionEvent::Response::Destroyed& event) {
                                    svc_response_rx_session_nodes_.removeNodeFor(event.service_id);
                                }),
                            event);
```

We should have used the same for `TxTransferHandler`, I overlooked this part in the previous PR.

Reply:
I agree about `SessionEventHandler` but applying the same for `TxTransferHandler` makes things quite ugly, like this:
<img width="1003" alt="image" src="https://github.com/OpenCyphal-Garage/libcyphal/assets/2915466/16a59086-5318-4b89-96a9-3c6f5707f1fa">

Reply:
Also, idea of these `XxxHandler`-s was to encapsulate handling business logic into small but separate "handler" private sub-entity. So, I'm not quite convinced that using `make_overloaded` is better here...